tests/unit/llms/test_llm_loader.py

```diff
@@ -44,7 +47,7 @@ def test_constructor_wrong_provider():
 
 
 @pytest.mark.parametrize("provider, model", llm_cfgs)
-def test_constructor_correct_provider_no_models(provider, model):
+def _test_constructor_correct_provider_no_models(provider, model):
```

Comment:
is the intent here to disable this test?  if it's being replaced by the new test why not just delete it?

Reply:
actually it was leftover :/